.github/workflows/golangci-lint.yml

```diff
@@ -23,4 +23,5 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v6
         with:
-          version: v1.60
\ No newline at end of file
+          version: v1.60
+          args: "--config=.golangci.yml"
```

Comment:
[SUGGESTION] It seems you don't need it.
https://golangci-lint.run/usage/configuration/#config-file